neusomatic/python/generate_dataset.py

```diff
@@ -629,25 +629,26 @@ def prepare_info_matrices_tabix(
     for i, v in col_pos_map.items():
         col_pos_map[i] -= sum(np.array(cols_to_del) < v)
 
-    tumor_matrix = np.delete(tumor_matrix_, cols_to_del, 1)
-    bq_tumor_matrix = np.delete(bq_tumor_matrix_, cols_to_del, 1)
-    mq_tumor_matrix = np.delete(mq_tumor_matrix_, cols_to_del, 1)
-    st_tumor_matrix = np.delete(st_tumor_matrix_, cols_to_del, 1)
-    lsc_tumor_matrix = np.delete(lsc_tumor_matrix_, cols_to_del, 1)
-    rsc_tumor_matrix = np.delete(rsc_tumor_matrix_, cols_to_del, 1)
+    cols_to_keep = list(set(range(n_col))-set(cols_to_del))
+    tumor_matrix = tumor_matrix_[:, cols_to_keep]
```

Comment:
Thanks @pavanstrand . Do you think these matrix generations can also be generalized? So, instead of having individual lines for each matrix, we iterate over a list of features for which we need to generate these matrices?

Reply:
I am working on that next which I mentioned as removing the redundancy.

Reply:
Sounds good!